app/src/main/java/com/sdp/movemeet/MainActivity.java

```diff
@@ -47,35 +111,11 @@ public void handleRegisterUser() {
             TextView[] textViewArray = {fullName, email, phone};
             FirebaseInteraction.retrieveDataFromFirebase(fStore, userId, textViewArray, MainActivity.this);
         }
-
-    }
-
-    public void goToMaps(View view) {
-        startActivity(new Intent(this, MapsActivity.class));
     }
 
 
     public void logout(View view) {
         FirebaseInteraction.logoutIfUserNull(fAuth, MainActivity.this);
```

Comment:
Yes, I'm soon going to rename this function from "logoutIfUserNull" to "logoutIfUserNotNull" (or simply "logout" 😂)

Reply:
Haha no worries, we will surely make a small refactor to use your function everywhere ;-)